### PR TITLE
[API-1170] Add `testing_type` to POST and PUT pentest endpoints

### PR DESCRIPTION
**Pull request by ericcornelissen**

### CODE REVIEW

File: versions/v2/content/pentests.md [hunk 1]
```diff
@@ -410,7 +411,8 @@ organization specified in the `X-Org-Token` header.  The pentest will be in the
 | `additional_requests`     | Optional; if present, special requests or instructions for the pentester(s).                                                                             |
 | `test_credentials`        | Optional; if present, credentials to be used by the pentester(s).                                                                                        |
 | `test_credentials_option` | Optional; if present, additional information about the credentials. Must be one of the following: `provided`, `distributed`, `not_required`              |
-| `targets`                 | Optional; if present, the URL(s), endpoint(s), etc. being targeted by the pentest. A comma separated list; `"1.1.1.1, https://cobalt.io"`                  |
+| `targets`                 | Optional; if present, the URL(s), endpoint(s), etc. being targeted by the pentest. A comma separated list; `"1.1.1.1, https://cobalt.io"`                |
+| `testing_type`            | Optional; If present, this will be the type of the pentest. Must be one of the following: `internal`, `agile`, `comprehensive`. Default: `comprehensive` |
```
Comment (inikolov89) [inline versions/v2/content/pentests.md:415]: Do we have some documentation so we can link it to better explain the difference between the testing types?

**Review by inikolov89 — APPROVED** [review]
LGTM. Just wondering if we should explain the different testing types. I am not sure how self-explanatory they are.

File: versions/v2/content/pentests.md (hunk 1 above)
Comment (ericcornelissen) [reply versions/v2/content/pentests.md:415]: Happy to add it, don't know about any such documentation.

Do note though that this is not the first introduction of testing types to the documentation, see https://github.com/cobalthq/cobalt-api-docs/pull/114.

**Review by zmomen-cobalt — APPROVED** [review]

File: versions/v2/content/pentests.md (hunk 1 above)
Comment (ericcornelissen) [reply versions/v2/content/pentests.md:415]: I'll merge this as is, we can add a link later once we have one 🙂

**Pull request merged**